### ISSUE TITLE
can you port a mod for me

### ISSUE THREAD

**Issue by fnfporterhi** [LEVIXDDLMAO/Tabi_rework_html5#1]
mod link: https://gamebanana.com/mods/329361

**Comment by LEVIXDDLMAO** [comment]
sorry this mod has no source code so i cant port it

**Comment by fnfporterhi** [comment]
ok what about this one: https://gamebanana.com/mods/334945

this has source code

**Comment by LEVIXDDLMAO** [comment]
Maybe if i do it, it will take a few days

**Comment by fnfporterhi** [comment]
Ok thanks

**Comment by LEVIXDDLMAO** [comment]
And go check out the Funked engine! :D

**Comment by fnfporterhi** [comment]
Do you have a html link yet? I wanna be a beta tester…

**Comment by fnfporterhi** [comment]
If that’s ok

**Comment by LEVIXDDLMAO** [comment]
So the early version of the game is not working right now so html is gonna be hold back untill beta (0.2) is out and working on windows and then maybe, the leather engine html5 is being done by one of my coders for the Funked engine So i will let you know when its done

**Comment by fnfporterhi** [comment]
Thanks dude :)

**Comment by LEVIXDDLMAO** [comment]
Np! Sorry if we dont get Funked engine html port to work we are doing the best we can

**Comment by fnfporterhi** [comment]
Oh it’s fine! No rush! I’m bored right now and got a lot of coding to do to unlock fps on HTML5 for chrome books…

**Comment by LEVIXDDLMAO** [comment]
thats cool!

**Comment by fnfporterhi** [comment]
Yeah, it is, had 1 successful run and got 335 fps the next minute it says Error: cannot import outside module…

**Comment by fnfporterhi** [comment]
But hey it’s the coding thing, gotta deal with haxe too lol

**Comment by LEVIXDDLMAO** [comment]
you have haxe?

**Comment by fnfporterhi** [comment]
Kinda, I have to have it to import the settings to unlock the fps limit, cause right now unless you have a 120 HZ refresh rate monitor, you’re stuck with 61 fps until I fix the import bug..

**Comment by LEVIXDDLMAO** [comment]
yeah So maybe when thats done i'll uploaded to the engine!

**Comment by fnfporterhi** [comment]
Welp I’m going to bed it’s 5:00 AM here..

**Comment by fnfporterhi** [comment]
Hey dude, I tested the engine (I’ve been watching the repo issue), and right now note colors and controls are the main problem but everything else works! Except for other songs, I can only play milf (which happens to be my favorite song), but still works great!

**Comment by LEVIXDDLMAO** [comment]
the beta(0.2) or newer (0.3)?

**Comment by fnfporterhi** [comment]
Beta

**Comment by fnfporterhi** [comment]
Haven’t seen newer

**Comment by LEVIXDDLMAO** [comment]
just wondering because 0.3 is not ported yet And (0.2) should work i testing it now

**Comment by fnfporterhi** [comment]
ok

**Comment by fnfporterhi** [comment]
Also update on the unlimited fps for chrome books: I tested it the other day, VERY buggy, and it doesn’t work half the time I tested it though on the vanilla game, but it is very buggy still, and I got a lot of work to do to fix it

**Comment by LEVIXDDLMAO** [comment]
oh  well take your time

**Comment by fnfporterhi** [comment]
FINALLY FIXED THE BUG! After 9 non-stop days of not sleeping, relying on coffee and other stuff, it works!

**Comment by LEVIXDDLMAO** [comment]
Cool!! how much fps do you get?

**Comment by fnfporterhi** [comment]
with a 75hz referesh rate monitor and the code, I get around 125-140

I tested it on the fnf megapack so that's not official yet, I've gotta test it on a version where it wont lag

**Comment by fnfporterhi** [comment]
ill send a recording in a minute

**Comment by LEVIXDDLMAO** [comment]
Okay! ^^

**Comment by fnfporterhi** [comment]
ok so recording doesnt like my chromebook, but here's the code

**Comment by fnfporterhi** [comment]
`openfl.display.BlendMode;
 openfl.text.TextFormat;
 openfl.display.Application;
 flixel.util.FlxColor;
 flixel.FlxG;
 flixel.FlxGame;
 flixel.FlxState;
 openfl.Assets;
 openfl.Lib;
openfl.display.FPS;
 openfl.display.Sprite;
openfl.events.Event;

var game:FlxGame;
//optimize this fuckin shit

#if switch
   apply();
#end

private function apply():Void
{
  #if !html5
	  	framerate = 100;
  #else
       framerate = 60;
   #end 

 
  //openfl.Lib.current.stage.frameRate = cap
  var framerate:Int = 0;
 } `

**Comment by fnfporterhi** [comment]
ive gotta fix a bug but the rest is good

**Comment by LEVIXDDLMAO** [comment]
What should the name be?
Take her time sorry it didn;t send

**Comment by fnfporterhi** [comment]
I think the name should be based off of the fps+ for windows, FPS+ chromebook edition

**Comment by LEVIXDDLMAO** [comment]
i mean the hx i might add it to the engine

**Comment by fnfporterhi** [comment]
Oh uhhhhhhhhhhhhhhhhh, I honestly don’t know lol

**Comment by fnfporterhi** [comment]
fps?

**Comment by LEVIXDDLMAO** [comment]
Yeah that would work!

**Comment by fnfporterhi** [comment]
Also I have worked on a side project besides this, I created a Fnf app for iPad basically

**Comment by fnfporterhi** [comment]
It’s not released publicly because it’s android and Apple supported.

**Comment by LEVIXDDLMAO** [comment]
Ooo!

**Comment by fnfporterhi** [comment]
I’ve been devoleping it, it has 35 weeks, including everyone sings (3 songs), tabi, sky, and more

it’s a gigabyte big so

**Comment by LEVIXDDLMAO** [comment]
Yeah the engine is big too

**Comment by fnfporterhi** [comment]
Lol same for mine I just had to reboot my iPad just to get it to finally boot up


I really need a new one mine is so old lol

Sending a preview as soon as it finishes downloading

**Comment by LEVIXDDLMAO** [comment]
Oh okay!
I cant wait!

**Comment by fnfporterhi** [comment]
DAMN YOU GITHUB FILE TO BIG

URL: https://share.icloud.com/photos/07cY268L6OfIo_7G1sAkWdn2Q

**Comment by LEVIXDDLMAO** [comment]
lmao-

**Comment by LEVIXDDLMAO** [comment]
Why dose this look like [FunkinForever](https://www.youtube.com/channel/UCdgNoVSF2wYJ3HRaISBTwGQ)

**Comment by fnfporterhi** [comment]
I have honestly never heard of funkin forever, lemme see if it’s on App Store, cause I’m interested to see how it looks like mine lmao

**Comment by fnfporterhi** [comment]
Now I see the resemblance. Weird

**Comment by LEVIXDDLMAO** [comment]
lol

**Comment by fnfporterhi** [comment]
I ran mine off of mic’d up engine with mobile support, more note skins (as shown in the video), and like a bunch of characters that are playable but one thing, I don’t have Dave and Bambi lol

**Comment by LEVIXDDLMAO** [comment]
Ah imma gonna need someone to port mic'd up enigne to html ;mao

**Comment by fnfporterhi** [comment]
I think I know a guy who can do it….

**Comment by LEVIXDDLMAO** [comment]
Who?

**Comment by fnfporterhi** [comment]
Someone on itch.io

**Comment by LEVIXDDLMAO** [comment]
Ooh I cant really get on itch.io-

**Comment by fnfporterhi** [comment]
Don’t worry I’ll send the html link that’s not itch io, what I do is I f12 it and use the link from the embed and just bookmark the link

**Comment by LEVIXDDLMAO** [comment]
Thx! And can you ask if he can do challenge-edd psych port?

**Comment by fnfporterhi** [comment]
Sure!!!

**Comment by LEVIXDDLMAO** [comment]
Thanks!

**Comment by fnfporterhi** [comment]
Done!!! He’s also porting aflac

**Comment by LEVIXDDLMAO** [comment]
Cool! the newer one right?

**Comment by fnfporterhi** [comment]
Yep

**Comment by LEVIXDDLMAO** [comment]
finally i wanna try the newer one so bad

**Comment by fnfporterhi** [comment]
same I tried Dominic’s port and I just get a black screen and a mouse that lags when you move it, no offense against him

**Comment by LEVIXDDLMAO** [comment]
Oh its fine

**Comment by fnfporterhi** [comment]
And I also have another guy on GitHub who won’t reply to any issues. Like there’s one port he did and it just does asset loading 2000 times and never goes to the intro screen, and I told him, no reply, again a month later, I asked him, no reply. I’m about to block him if he doesn’t reply lol

**Comment by fnfporterhi** [comment]
Damn it. I gotta go, my mom is telling me to “go outside” so I gotta bring my laptop…

**Comment by LEVIXDDLMAO** [comment]
Ah okay! ill be here

**Comment by LEVIXDDLMAO** [comment]
@FNFUNBLOCKED  you there?

**Comment by fnfporterhi** [comment]
Good morning! Sorry I was asleep it was 10:00 my time, the guy replied and said he made his own version of it… want me to send link?

**Comment by fnfporterhi** [comment]
Also he’s finishing Aflac then porting indie cross..

**Comment by fnfporterhi** [comment]
What time zone are you in? I’m in EST (Eastern Standard Time)

**Comment by fnfporterhi** [comment]
My school has testing today so just a little note: I may not reply immediately but give me until 11:25 my time to reply, I get my phone back then..

**Comment by fnfporterhi** [comment]
Unless it’s critical just let me jnow

**Comment by fnfporterhi** [comment]
Know

**Comment by LEVIXDDLMAO** [comment]
oh okay

**Comment by fnfporterhi** [comment]
Bsck

**Comment by fnfporterhi** [comment]
I’m back

**Comment by fnfporterhi** [comment]
Sorry I had my phone about replied to you and it died

**Comment by LEVIXDDLMAO** [comment]
its fine but did that ported get the things done

**Comment by fnfporterhi** [comment]
Not yet, he’s still working on it, he never finished it, but the attacks are what’s going to be a bit… but besides that yea.

**Comment by LEVIXDDLMAO** [comment]
What are the ports that are done?

**Comment by fnfporterhi** [comment]
CHALLENGEEDD DONE!: https://pizzashowerman.itch.io/funkin-eddnuberkid-psych

**Comment by LEVIXDDLMAO** [comment]
I cant use itch.io lol

**Comment by fnfporterhi** [comment]
SORRY!! I’ll port when I get home. Just got off school.

**Comment by fnfporterhi** [comment]
Not port get link

**Comment by LEVIXDDLMAO** [comment]
lol its fine Im in school rn anyways

**Comment by LEVIXDDLMAO** [comment]
Just tell me when your home

**Comment by LEVIXDDLMAO** [comment]
@FNFUNBLOCKED  You home?

**Comment by fnfporterhi** [comment]
Yep sorry I had to do something, app issue, and I’ll fix link in a min lemme get my pc turned on (time to wait 10 years cause of all the stuff that’s on there)

**Comment by fnfporterhi** [comment]
HOOOOOOOLLLLYYYYY SSSHHHHIIITTTT LEVI I JUST FOUND GRAVITY FALLS OVER WEEK 6 FNF MOD


sorry I just love the show

Also getting link now

**Comment by LEVIXDDLMAO** [comment]
SAMEEEEE BRO LOL

**Comment by LEVIXDDLMAO** [comment]
SORRY I JUST BACK LOL @FNFUNBLOCKED

**Comment by LEVIXDDLMAO** [comment]
@FNFUNBLOCKED  you there?

**Comment by fnfporterhi** [comment]
Time difference remember?

**Comment by fnfporterhi** [comment]
Also GitHub isn’t blocked at my school, so can I help with the engine and stuff? Saw that y’all had problems with that.

**Comment by fnfporterhi** [comment]
What state are you in cause I’m in Tennessee

**Comment by fnfporterhi** [comment]
I ask that only wanting to know what’s the time difference between us

**Comment by LEVIXDDLMAO** [comment]
im in denver lol

**Comment by fnfporterhi** [comment]
Morning! I’ll get the link to you later testing has ended at school but still I may be slow to respond.

**Comment by LEVIXDDLMAO** [comment]
Morning! ^^ its fine

**Comment by fnfporterhi** [comment]
https://v6p9d9t4.ssl.hwcdn.net/html/5181293/bin/index.html theres challenge edd

**Comment by LEVIXDDLMAO** [comment]
i

**Comment by LEVIXDDLMAO** [comment]
its blocked lol

**Comment by LEVIXDDLMAO** [comment]
its blocked lol

**Comment by fnfporterhi** [comment]
uh oh

**Comment by fnfporterhi** [comment]
That’s my only source of access… damn

**Comment by LEVIXDDLMAO** [comment]
No im trying something rn

**Comment by LEVIXDDLMAO** [comment]
that didn't work lol

**Comment by LEVIXDDLMAO** [comment]
can you ask him for the all the files on google drive i can work with that lol

**Comment by fnfporterhi** [comment]
yea, also I did that cause my school gives us surface laptops instead of chromebooks lol

**Comment by LEVIXDDLMAO** [comment]
Thx, we get crappy chromebook

**Comment by fnfporterhi** [comment]
damn I understand that through 5-7th grade before my school got a LOT of money from some rich people who died they gave us school chromebook with 8GB memory

**Comment by LEVIXDDLMAO** [comment]
XD
